.github/workflows/test.yml

```
@@ -11,6 +11,25 @@ on:
       - synchronize
 
 jobs:
+  Mac:
+    name: Test Mac
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v1
```

Nitpick, just to make it consistent with windows/linux
```suggestion
      - name: Checkout branch
        uses: actions/checkout@v1
```